src/components/Card.spec.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react';
+import PlantCard from './Card';
+
+describe('PlantCard', () => {
+  it('renders a list of cards with information about plants', () => {
+    render(<PlantCard />);
+
+    const PlantCards = screen.getByRole('list');
+    expect(PlantCards).toBeInTheDocument();
+  });
```

Comment:
Currently you are testing if the list is in your component.
You should also test if the information about your plant is displayed. 
(try `getByText`  https://testing-library.com/docs/queries/about/#types-of-queries)

Reply:
Thank you, I fixed your suggestions :)